$OpenBSD: patch-jsfun_cpp,v 1.1 2018/04/08 10:53:10 landry Exp $

jsfun.cpp:372:34: error: assigning to 'jsuword' (aka 'unsigned long') from incompatible type 'nullptr_t'
    wfun->u.i.names.taggedAtom = NULL;  

Index: jsfun.cpp
--- jsfun.cpp.orig
+++ jsfun.cpp
@@ -369,7 +369,7 @@ WrapEscapingClosure(JSContext *cx, JSStackFrame *fp, J
     wfun->u.i.skipmin = fun->u.i.skipmin;
     wfun->u.i.wrapper = true;
     wfun->u.i.script = NULL;
-    wfun->u.i.names.taggedAtom = NULL;
+    wfun->u.i.names.taggedAtom = 0;
     wfun->atom = fun->atom;
 
     if (fun->hasLocalNames()) {
